Add Latest Presentations + Document Knowledge Sharing Session(s)

jafeltra opened the pull request:
**Description:**
This PR adds the latest DevDays FSH presentations to the Downloads page.  I'm not aware of other presentations we've done recently, but if there are others besides DevDays to include, let me know.

I also added a section with links to the recording and slides of the Community Kickoff Webinar and all previous Knowledge Sharing Sessions. While FSH School is typically focused on documentation for users of the FSH tools, the KSS presentations are typically helpful for developers of FSH tools. Let me know if you think the section is out of place, and I can remove it.

While I was in this repo, I also added the link to the Community Engagement, Issue Tracking, and Documentation session in the README.

**Related Issue:** N/A

cmoesel commented on content/downloads/_index.md:
```diff
@@ -35,15 +35,46 @@ For full instructions and prerequisites, see [GoFSH Installation](/docs/gofsh/in
-* <a href="https://youtu.be/Ohgx5TedLlI">Advanced FHIR Shorthand and Tools</a>, from FHIR DevDays, June 2021
-  * Slides <a href="20210610_Chris_Moesel_Advanced_FSH_And_Tools.pdf">here</a>
+* <a href="https://youtu.be/tmtEnjxrCDA">Learn to FSH: A friendly introduction to FHIR Shorthand</a>, from FHIR DevDays, June 2023
+  * Slides <a href="230606_MarkKramer_Learn_to_FSH.pdf">here</a>
```
Is there a reason you have uploaded these presentations to FSH School instead of directly linking to them on the DevDays site? For example, this presentation is available here: https://www.devdays.com/wp-content/uploads/2023/08/230606_MarkKramer_Learn_to_FSH.pdf

jafeltra replied:
I uploaded the presentations just to align with the other presentations that were already there. I'm not opposed to using the direct links for all of them though.

jafeltra replied:
I updated all the DevDays presentations to link to the presentation on the DevDays site.

However, I couldn't find the Let's Build slides from June 2021 on the DevDays site. There were two sets of the slides from the Create an IG with FSH presentation from that year, so I have a feeling that presentation got uploaded twice and the Let's Build didn't get uploaded. Since we had the slides uploaded to FSH School already, I kept those for that one presentation.

cmoesel commented on content/downloads/_index.md:
```diff
@@ -35,15 +35,46 @@ For full instructions and prerequisites, see [GoFSH Installation](/docs/gofsh/in
+* <a href="https://youtu.be/u6Btqb0Sz3M">Advanced FSH: What to do when you’re hooked</a>, from FHIR DevDays, June 2023
+  * Slides <a href="230609_ChrisMoesel_AdvancedFSH.pdf">here</a>
+* Let's Build - Create an Implementation Guide with FHIR Shorthand, from FHIR DevDays, June 2022
+  * Slides <a href="220607-ChrisMoesel-CreateAnIGWithFHIRShorthand-Chris-Moesel.pdf">here</a>
```
Mark Kramer had a presentation in 2022 as well:
* video: https://youtu.be/jegp6qEjgwM?si=rt0Ce-rWx4Tiel_o
* slides: https://www.devdays.com/wp-content/uploads/2022/08/220607_MarkKramer_Learn_to_FSH-Mark-Kramer.pdf

cmoesel commented on content/downloads/_index.md:
```diff
@@ -35,15 +35,46 @@ For full instructions and prerequisites, see [GoFSH Installation](/docs/gofsh/in
+* <a href="https://youtu.be/RfmqpUA606U">Introduction to FHIR Shorthand</a>, from FHIR DevDays, November 2020
+  * Slides <a href="201118_Mark_Kramer_Introduction_To_FHIR_Shorthand.pdf">here</a>
+* <a href="https://youtu.be/7yzLzQjict0">Let's Build - Profiling with FHIR Shorthand</a>, from FHIR DevDays, November 2020
+  * Slides <a href="201118_Chris_Moesel_Lets_Build_with_FHIR_Shorthand.pdf">here</a>
```
If you want to be historically complete, there is also:
* FHIR Shorthand, from FHIR DevDays, June 2020
  * video: https://youtu.be/oK3-wYzJeIA?si=i5WjBymhC9zeFePc
  * _slides not available_
* Let's Build! FHIR Shorthand, from FHIR DevDays, June 2020
  * video: https://youtu.be/iJGmHiAlQwo?si=kwK5VMaRTH0azBBs
  * slides: https://www.devdays.com/wp-content/uploads/2021/12/Lets-Build-FHIR-Shorthand-DevDays-2020-Virtual.pdf
* FHIR Shorthand, from FHIR DevDays, November 2019
  * slides: https://www.devdays.com/wp-content/uploads/2021/12/Mark-Kramer-FHIR-Shorthand-_-DevDays-2019-Amsterdam-1.pdf
  * video: https://youtu.be/VGUEeQSd6AM?si=zKo7zJyrMfo01U4u

jafeltra replied:
I think I want to be historically complete. But I guess the history is longer than I had originally thought. Do we feel like this provides too much repetition?

jafeltra replied:
After discussing it, we like the complete historic list. So I included these as well.

cmoesel commented on content/downloads/_index.md:
```diff
@@ -35,15 +35,46 @@ For full instructions and prerequisites, see [GoFSH Installation](/docs/gofsh/in
+### HL7 Community Kickoff and Knowledge Sharing Sessions
+
+The [FSH Tooling Community Kick Off Webinar](https://hl7-org.zoom.us/rec/share/E-adnsXQFT2urCZw3ERdM_HMCXl1PSyBsVwX3h-_SZe9D8uC415UObFdrERYEpa1.av4D1wJRJSCOTmCE) from August 2024 discussed the current state of FHIR Shorthand and its associated tools, the governance of the community-driven project going forward, and possible next steps for the project.
+The [slides](https://confluence.hl7.org/display/FHIR/FSH+Tooling+Community+Kick+Off+Webinar?preview=/265100106/265100122/FSH%20Tooling%20Community%20Kickoff.pdf) can be found on Confluence.
```
I noticed that for Confluence, you have linked to the previews rather than the direct PDF downloads. Is that intentional?

jafeltra replied:
This feels a little counter intuitive based on my other answer above, but since it was a new section, I didn't mind having the links to Confluence, rather than uploading the PDFs to FSH School. However, the preview was the only link I could find to share the PDF on Confluence. Is there a better way to link to a PDF from Confluence? Or do you prefer the direct download from FSH School?

jafeltra replied:
After talking about the options we have for linking to PDFs on Confluence, we decided to keep these preview links as is.

cmoesel reviewed (changes requested):
It's great to see this all here! I found a few that you missed and also asked a couple of questions...

jafeltra commented:
@cmoesel -- I believe [2ea0190](https://github.com/FHIR/FSHSchool/pull/107/commits/2ea01907bf09da4cd9923e71e4ce16da54a3d5bf) addresses all the comments you had. I did look through old versions of our presentation list, and I didn't find the slides for the June 2020 presentation, so I left those slides as unavailable. As mentioned above, I also couldn't find the slides from June 2021 on the DevDays site, so those slides are still uploaded to FSH School.

Let me know what you think.

cmoesel reviewed (approved):
Thanks, Julia! I've checked all the links and they look great. It's cool to be able to look back and see how FSH was presented in the early days!
